engine/battle/effects.asm

```diff
@@ -118,7 +118,7 @@ PoisonEffect:
 	ret nc
 .inflictPoison
 	dec hl
-	set 3, [hl] ; mon is now poisoned
+	set PSN, [hl] ; mon is now poisoned
```

Comment:
We would typically get rid of the comment at this point, since the usage of the constant accomplishes the same thing but better.